src/fmgc/src/flightplanning/ManagedFlightPlan.ts

```diff
@@ -482,28 +481,21 @@ export class ManagedFlightPlan {
      * @param index The index of the waypoint to remove.
      */
     public removeWaypoint(index: number, noDiscontinuity: boolean = false): void {
-        let removed = null;
         if (this.originAirfield && index === 0) {
-            removed = this.originAirfield;
             this.originAirfield = undefined;
-
             this.reflowSegments();
             this.reflowDistances();
         } else if (this.destinationAirfield && index === this.length - 1) {
-            removed = this.destinationAirfield;
             this.destinationAirfield = undefined;
         } else {
             const segment = this.findSegmentByWaypointIndex(index);
             if (segment) {
-                // console.log("--> REMOVING WAYPOINT ", this.getWaypoint(index), ", FROM SEGMENT ", segment);
                 const spliced = segment.waypoints.splice(index - segment.offset, 1);
-                removed = spliced[0];
-
+                console.log(`removing waypoint ${spliced[0].icao} from segment ${segment.type}`);
```

Comment:
Comment out Debug?

Reply:
Would like to keep this one to easier debug issue when users report problems.